examples/sketch.rs

```diff
@@ -1,6 +1,11 @@
 use ethereum_consensus::altair::light_client::Snapshot;
+use ethereum_consensus::phase0::mainnet::{apply_block, BeaconState, SignedBeaconBlock};
 
 fn main() {
+    let state = BeaconState::default();
+    let signed_block = SignedBeaconBlock::default();
+    let _post = apply_block(state, signed_block);
```

Comment:
example usage: all generic bounds have been specified by the time an item is exposed in its respective `preset`.